addon/components/column-chart/component.js

```diff
@@ -95,13 +99,25 @@ export default Ember.Component.extend({
 
         const xAxis = this.get('xAxis');
         const yAxis = this.get('yAxis');
+        const xTimeScale = d3.time.scale().domain(xAxis.domain);
+
+        const indicatorDate = this.get('currentInterval') ? this.get('currentInterval.start._d') : null;
+        const showCurrentIndicator = this.get('showCurrentIndicator');
 
         const titles = this.get('series').map(entry => entry.title);
 
         const data = this.get('data');
         const tooltipDateFormat = this.get('tooltipDateFormat');
         const formatter = this.get('xAxis.formatter') || (value => value);
 
+        function isIntervalIncluded(ticks, interval) {
+            return ticks.toString().includes(interval.toString());
+        }
+
+        function isIntervalInRange(scale, interval) {
+            return (scale.ticks().pop() >= interval && scale.ticks()[0] <= interval);
```

Comment:
Outer parens are not needed. And, `pop` will modify the underlying array, not sure if that's a problem here or not though.

Reply:
Modification of array shouldn't matter, as scales don't retain changes. This was just the simplest way to grab the last tick in the scale

Reply:
Ok cool that's what I figured :+1: